sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/implementation/FilesImpl.java

```diff
@@ -412,7 +413,7 @@ public FilesImpl(AzureFileStorageImpl client) {
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<FilesUploadRangeResponse> uploadRangeWithRestResponseAsync(String shareName, String filePath, String range, FileRangeWriteType fileRangeWrite, long contentLength, Context context) {
-        final Flux<ByteBuf> optionalbody = null;
```

Comment:
Do we need to update the swagger as well?

Reply:
@jianghaolu will need to ensure autorest generates code with ByteBuffer rather than ByteBuf, but there will not be any changes made to the swagger because of this.